### ISSUE TITLE
keeweb-native-messaging-host doesn't start on Ubuntu 18.04

### ISSUE THREAD

**Issue by synchrone** [keeweb/keeweb-connect#1]
```
$ apt-cache show libc6 | grep Version
Version: 2.27-3ubuntu1.4
Version: 2.27-3ubuntu1.2
Version: 2.27-3ubuntu1

$ uname -a
Linux syn-desktop 5.4.0-72-generic #80~18.04.1-Ubuntu SMP Mon Apr 12 23:26:25 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

$ ./keeweb-native-messaging-host 
./keeweb-native-messaging-host: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by ./keeweb-native-messaging-host)
./keeweb-native-messaging-host: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by ./keeweb-native-messaging-host)
```

**Comment by antelle** [comment]
Hi! What if you upgrade it to 2.28+?

**Comment by synchrone** [comment]
I would have to switch to another release branch (from bionic aka 18.04 to focal, aka 20.04)
https://packages.ubuntu.com/bionic-updates/libc6 -> https://packages.ubuntu.com/focal/libc6

Current 18.04 release branch is [supported until 2023](https://ubuntu.com/about/release-cycle), and I would like to stick to it.

**Comment by antelle** [comment]
Does this thing run? https://github.com/keeweb/keeweb-connect/suites/2610860888/artifacts/57162694

**Comment by synchrone** [comment]
yes

**Comment by antelle** [comment]
Great thanks, I'll rebuild it on v1.18 then.  
But although it runs, it won't work, because I commented out some code, so don't use it please... A fix it coming...

**Comment by antelle** [comment]
Fixed it, here's a build for Ubuntu 18.04: https://github.com/keeweb/keeweb-connect/releases/latest
